components/aws/sagemaker/hyperparameter_tuning/component.yaml

```diff
@@ -121,6 +145,14 @@ implementation:
       --continuous_parameters, {inputValue: continuous_parameters},
       --categorical_parameters, {inputValue: categorical_parameters},
       --channels, {inputValue: channels},
+      --data_location_1, {inputValue: data_location_1},
```

Comment:
Where are these values coming from _1 to _8? I believe for number of channels ? 
so number of channels can be at most 9?

Reply:
Yeah there's a limit of maximum 8 channels for HPO/training, so that's why I have channels 1-8 and it just corresponds to channel configs 1-8 when you're specifying the channels param

Reply:
actually I just checked again, I think it got increased to max 20

Reply:
better add a comment on the source where you are checking probably some documentation. 
It would be nice someone can later revisit this and gets the source of truth as well.

Reply:
true - I couldn't find documentation that says this though. it only says in SM, in the create training job form

Reply:
true - I couldnt find documentation that actually says this though. I found that info on the actual create hpo/training job form in SM